source/common/network/connection_impl.cc

```diff
@@ -541,7 +615,7 @@ void ConnectionImpl::onReadReady() {
 
 absl::optional<Connection::UnixDomainSocketPeerCredentials>
 ConnectionImpl::unixSocketPeerCredentials() const {
-  // TODO(snowp): Support non-linux platforms.
+// TODO(snowp): Support non-linux platforms.
```

Comment:
Is this merge error?

Reply:
@yanavlasov this is a format fix my patch picked up. Do you suggest I revert this?

I raised it this issue earlier.

Reply:
I just did a merge with latest master and see this file:

root@rkdev-vm-2997079:/home/rkavuluru/envoy# git status
On branch network_socket_options
Your branch is ahead of 'rk/network_socket_options' by 49 commits.
  (use "git push" to publish your local commits)
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	tools/.aspell.en.pws

This is a file I have not touched. I can add, commit and push again. I dont think the conflict was because of the above change.

Reply:
Sorry, I take that back. I think it was a generated spell check file (may be? ). I deleted it, pulled from origin master and merge my branch to master. Do not see any merge conflicts. I am leaving this as is.

Please let me know your thoughts if you need me to change something.

Thanks!